src/lib/CommandHandler.ts

```diff
@@ -47,11 +47,38 @@ export class CommandHandler {
         this.input = this.resolveTaskToInput(args.taskId);
 
         this.userInputContext = userInputContext;
-        this.args = args;
+        this.args = this.resolveBooleanArgs(args);
         this.context = context;
         this.subprocess = subprocess;
     }
 
+    protected resolveBooleanArgs(args: ShellCommandOptions): ShellCommandOptions {
```

Comment:
I think the type for `args` is also `unknown` (same for the constructor). You can move `this.args = this.resolveBooleanArgs(args);` to the top of the constructor and then read from `this.args` after that to make typescript happy.

Reply:
use `object` for `args` , seems more proper? 
and moved `resolveBooleanArgs` to the top and replaced all the references to `args` as `this.args`

Reply:
`object` sounds good